### ISSUE TITLE
DB table creation errors

### ISSUE THREAD

**Issue by jeremiahwinsley** [arobbins/wp-shopify#50]
Plugin Version: 1.0.12
PHP Version: 7.0.18
Wordpress Version: 4.8.0

When I activated the plugin, the database table creation failed due to conflicting charsets.

Error message:

```
WordPress database error Conflicting declarations: 'CHARACTER SET DEFAULT' and 'CHARACTER SET utf8mb4' for query CREATE TABLE `wp_wps_settings_general` (
      `id` bigint(100) NOT NULL AUTO_INCREMENT,
                  `url_products` varchar(100) NOT NULL DEFAULT 'products',
                  `url_collections` varchar(100) NOT NULL DEFAULT 'collections',
      `url_webhooks` varchar(100) NOT NULL DEFAULT '(removed)',
      `num_posts` bigint(100) DEFAULT NULL,
      `styles_all` tinyint(1) DEFAULT 1,
      `styles_core` tinyint(1) DEFAULT 0,
      `styles_grid` tinyint(1) DEFAULT 0,
      `plugin_name` varchar(100) NOT NULL DEFAULT 'WP Shopify',
      `plugin_textdomain` varchar(100) NOT NULL DEFAULT 'wps',
      `plugin_version` varchar(100) NOT NULL DEFAULT '1.0.12',
      `plugin_author` varchar(100) NOT NULL DEFAULT 'Andrew Robbins',
      `price_with_currency` tinyint(1) DEFAULT 0,
                  PRIMARY KEY  (`id`)
                ) ENGINE=InnoDB DEFAULT CHARSET=DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_c...
```

Tracked this here: (same problem across all the database classes)

https://github.com/arobbins/wp-shopify/blob/fcb3b8918281a1e7fcf77a0c5876691a18a141f6/classes/db/class-db-settings-general.php#L214

According to https://developer.wordpress.org/reference/classes/wpdb/get_charset_collate, `$collate` should contain the whole `CHARSET` line at this point.

 I replaced these lines with `) ENGINE=InnoDB $collate";` and was able to complete the installation.

**Comment by arobbins** [comment]
Thanks @jeremiahwinsley I have this updated and plan on releasing in the next day or so.

**Comment by arobbins** [comment]
Fixed in a4f3f3beaec454c311bf1b62f305314b02f7237a
